src/components/LetterGrid/letterGrid.module.scss

```diff
@@ -0,0 +1,93 @@
+@import '../../styles/mediaQueries.scss';
+@import '../../styles/colors.scss';
+
+.gridContainer {
+  background-color: rgb(151, 137, 45);
+  width: 320px;
+  margin: auto;
+  display: grid;
+  grid-template-rows: repeat(5, 1fr);
+  grid-template-columns: repeat(3, 1fr);
+
+  .top,
+  .topLeft,
+  .topRight {
+    grid-row: -4/1;
+  }
+
+  .top {
+    /* grid-row: 1; */
+    grid-column: 2;
+    margin-bottom: 30px;
+  }
+
+  .topLeft {
+    /*  grid-row: 2; */
+    grid-column: 1;
+    align-self: flex-end;
+  }
+
+  .topRight {
+    /*    grid-row: 2; */
+    grid-column: 3;
+    align-self: flex-end;
+  }
+
+  .center {
+    /*  grid-row: 3; */
+    grid-column: 2;
+  }
+
+  .bottomLeft,
+  .bottomRight,
+  .bottom {
+    grid-row: -1/4;
+  }
+
+  .bottomLeft {
+    /* grid-row: 4; */
+    grid-column: 1;
+    align-self: flex-start;
+  }
+
+  .bottomRight {
+    /*  grid-row: 4; */
+    grid-column: 3;
+    align-self: flex-start;
+  }
+
+  .bottom {
+    /*  grid-row: 5; */
+    grid-column: 2;
+    align-self: center;
+    margin-top: 30px;
+  }
+
+  .flex {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .letter {
+    width: 4.5rem;
+    height: 4.5rem;
+    font-size: 2rem;
+    font-weight: 600;
+    border-radius: 100%;
+  }
+
+  .mainLetter {
+    color: $white;
+    background-color: $ocher-90;
+    font-size: 2.8rem;
+  }
+
+  .secondaryLetters {
+    background-color: $white;
```

Comment:
Save a class by putting the background-color of secondary letter on letter. mainLetter can the overwrite the background from letter.